core/arxiv/submission/services/stream/stream.py

```diff
@@ -0,0 +1,126 @@
+from typing import Optional
+
+import boto3
+from botocore.exceptions import ClientError
+from retry import retry
+
+from arxiv.integration.meta import MetaIntegration
+from arxiv.base import logging
+from arxiv.base.globals import get_application_config, get_application_global
+
+from ...domain import Submission, Event
+from ...serializer import dumps
+
+logger = logging.getLogger(__name__)
+
+
+class StreamPublisher(metaclass=MetaIntegration):
```

Comment:
It would be good to have a little documentation on this class.

Reply:
https://github.com/arXiv/arxiv-submission-core/issues/66